mods/gen4/formats-data.js

```diff
@@ -1664,7 +1664,7 @@ exports.BattleFormatsData = {
 	},
 	registeel: {
 		inherit: true,
-		randomBattleMoves: ["stealthrock", "ironhead", "curse", "rest", "sleeptalk", "thunderwave", "toxic"],
+		randomBattleMoves: ["stealthrock", "ironhead", "curse", "rest", "sleeptalk", "thunderwave", "toxic", "protect", "seismictoss"],
```

Comment:
This doesn't look too good. A lot of moves there are incompatible with each other. Curse and SR/Toss, RestTalk and SR/Wave/Toxic, Iron Head and Toss, Wave and Toxic. This is just asking for bad sets.